dev/watch.js

```diff
@@ -96,6 +97,9 @@ chokidar.watch(`${sassDir}/**/*.scss`).on('change', changedFile => {
                 filesToCompile.map(file => file.replace('scss', 'css'))
             );
         })
+        .then(() => {
+            copyFiles.task();
```

Comment:
Yep, that's not a bad idea - the only issue is that in dev we'll sometimes want the inline style for testing since email clients don't support external stylesheets. Maybe I could add an `inline-email-css` switch for this?